src/app/item-page/simple/item-types/untyped-item/untyped-item.component.html

```diff
@@ -65,6 +65,15 @@
       [fields]="['dc.identifier.citation']"
       [label]="'item.page.citation'">
     </ds-generic-item-page-field>
+    <!-- Below is an example of how to render one or more lat/lng points and/or bounding box rectangles
+         in a tiled map viewer. Set 'cluster' to true for marker clustering -->
+<!--    <ds-geospatial-item-page-field [item]="object"-->
+<!--                                   [label]="'item.page.places'"-->
+<!--                                   [pointFields]="['dcterms.spatial']"-->
+<!--                                   [bboxFields]="['external.spatial.bbox']"-->
+<!--                                   [cluster]="true"-->
+<!--                                >-->
+<!--    </ds-geospatial-item-page-field>-->
```

Comment:
Same comment as above.  I'd prefer if this was more dynamic, so you don't require people to modify the HTML code in order to enable it.  But, maybe I'm misunderstanding this?

Reply:
(answered above, tbc)